pkgs/development/interpreters/erlang/generic-builder.nix

```diff
@@ -98,6 +98,8 @@ stdenv.mkDerivation ({
 
   inherit src version;
 
+  LANG = "C.UTF-8";
+
```

Comment:
Without this `ex_doc` behaves very strangely. I don't see any issue with setting this always?

Reply:
FWIW, Elixir sets this always too, so this change makes it in line with Elixir: https://github.com/NixOS/nixpkgs/blob/83bc01a59c5d644a78bbb70c07ca26b3f1bf58f9/pkgs/development/interpreters/elixir/generic-builder.nix#L37-L38